mkosi/installer/apt.py

```diff
@@ -103,6 +103,5 @@ def invoke_apt(
     packages: Sequence[str] = (),
     apivfs: bool = True,
 ) -> None:
-    bwrap(apt_cmd(state, command) + [operation, *packages],
-          apivfs=state.root if apivfs else None,
-          env=state.config.environment)
+    cmd = apivfs_cmd(state.root) if apivfs else []
+    bwrap(cmd + apt_cmd(state, command) + [operation, *packages], env=state.config.environment)
```

Comment:
We're sorting package names for dnf and pacman, but not for apt and zypper, I think we should too.